articles/annotation_and_query.md

```diff
@@ -13,98 +13,85 @@ category: howto
+Annotations are key-value pairs stored as metadata for `Projects`, `Files`, `Folders`, and `Tables` that help users to find and query data. Annotations can be based on an existing ontology or controlled vocabulary, or can be created in an <i>ad hoc</i> manner and modified later as the metadata evolves. Annotations can be a powerful tool used to systematically group and/or describe things (files or data, etc.) in Synapse, which then provides a way for those things to be searched for and discovered.
 
-Annotations are structured as key-value pairs: the key is the name of the annotation while the value provides specifics. For example, if you have uploaded a collection of alignment files in the BAM file format from an RNA sequencing experiment, each representing a sample and experimental replicate, you can use annotations to add this information to each file in a structured way. Sometimes, users encode this information in file names, e.g., `Sample1_ConditionA.bam`, which makes it "human-readable" but makes it difficult to search for in a systematic way, such as finding all replicates of `Sample1`. Adding this information as Synapse annotations enables a more complete description of the contents of the `File`. 
 
-In this case, the annotations you may want to add might look like this: 
+ For example, if you have uploaded a collection of alignment files in the BAM file format from an RNA-sequencing experiment, each representing a sample and experimental replicate, you can use annotations to add this information to each file in a structured way. Sometimes, users encode this information in file names, e.g., `sampleA_conditionB.bam`, which makes it "human-readable" but makes it difficult to search for in a systematic way, such as finding all replicates of `sampleA_conditionB`. Adding this information as Synapse annotations enables a more complete description of the contents of the `File`. 
 
-* `assay = RNA-seq`
-* `fileType = bam`
-* `sample = 1`
-* `condition = A`
+ In this case, the annotations you may want to add might look like this: 
 
-For the second sample in the same experimental set with the same conditions, the annotations would have the same "keys" but slightly different "values", such as:
-
-* `assay = RNA-seq`
-* `fileType = bam`
-* `sample = 2` (note that we've changed the value here but the rest are the same)
-* `condition = A`
-
- If these samples were part of a dataset with multiple assays, such as RNA-seq and ATAC-seq, you would annotate the file entities with either `assay = RNA-seq` or `assay = ATAC-seq`. 
+ <img src="/assets/images/annotationsComplete.png">
 
 ## Types of Annotations
 
-Annotations can be one of four basic types 
+Annotations can be one of four basic types: 
 
 * Text (Character Limit = 256)
 * Integer
 * Floating Point
 * Date (Date and Time stored as a Timestamp)
 
+<br/>
 You cannot have duplicate annotation keys on the same object in Synapse. 
```

Comment:
maybe remove this? I don't understand what this means (@meredithslota, @kimyen?)

Reply:
Let me give an example to see if it helps:
Annotations are key-value pairs - stated above.
Annotations belong to an object (File, Project ...)
Annotation values can have one of the type mentioned above.
However, one cannot define an annotation key "Test" of any type while there is another annotation key "Test" of type Text already exist for the object: https://github.com/Sage-Bionetworks/Synapse-Repository-Services/blob/f7a1b539c451ba4c1f83177a3e86cbfc0218a326/lib/models/src/main/java/org/sagebionetworks/repo/model/annotation/AnnotationsUtils.java#L17

Reply:
OK, so it means the obvious thing :P I guess that's what I mean - why call it out, when it's basically what someone would expect?